pkg/odo/cli/service/create.go

```diff
@@ -167,25 +194,30 @@ func (o *ServiceCreateOptions) Validate() (err error) {
 			if err != nil {
 				return err
 			}
-			return fmt.Errorf("plan %s is invalid for service %v.\nPlease select one of: %v\n", o.plan, o.serviceType, strings.Join(ui.GetServicePlanNames(plans), ","))
+			return fmt.Errorf("plan %s is invalid for service %v.\nPlease select one of: %v\n", o.Plan, o.ServiceType, strings.Join(ui.GetServicePlanNames(plans), ","))
 		}
 	}
 	//validate service name
-	return o.validateServiceName(o.serviceName)
+	return o.validateServiceName(o.ServiceName)
 }
 
 // Run contains the logic for the odo service create command
 func (o *ServiceCreateOptions) Run() (err error) {
-	err = svc.CreateService(o.Client, o.serviceName, o.serviceType, o.plan, o.parametersMap, o.Application)
-	log.Successf(`Service '%s' was created`, o.serviceName)
+	err = svc.CreateService(o.Client, o.ServiceName, o.ServiceType, o.Plan, o.ParametersMap, o.Application)
+	log.Successf(`Service '%s' was created`, o.ServiceName)
 	log.Info(`Progress of the provisioning will not be reported and might take a long time.
 You can see the current status by executing 'odo service list'`)
+	equivalent := o.outputNonInteractiveEquivalent()
+	if len(equivalent) > 0 {
+		log.Info("Equivalent command:\n" + ui.StyledOutput(equivalent, "cyan"))
```

Comment:
The idea is not so much that it's colored but rather that it's visually different from the rest of the output so that users can see it more easily. Also, the color system uses the same one used by `survey` so if you see colored output already for selected answers to questions or question marks in the interactive mode, then it should work just as well.